scripts/package_policies.sh

```diff
@@ -0,0 +1,69 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/bin/bash
+
+set -e
+
+# https://github.com/forseti-security/policy-library repo constants
+COMMIT=f30bf774fb914959a9a18c132947eae95543001e
+SHA256=df40ad1133c3d48e9c34066b03d93ee08a15f134c8dcdce750ba36ba06cb4c84
+
+while getopts ":s:d:" opt; do
+  case $opt in
+    s)
+      src="$OPTARG"
+    ;;
+    d)
+      dst="$OPTARG"
+    ;;
+    \?)
+        echo "Invalid option -$OPTARG"
+        echo "usage: [-s source path] [-d destination path (use gs:// prefix for GCS bucket)]"
```

Comment:
Nit: I think it's best practice to use single quotes `''` when there are no variable substitutions.

Reply:
Done